src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -650,7 +649,7 @@ impl Node {
             layer_on_not_hover <- preview_visible.on_true().map(|_| visualization::Layer::Front);
             layer              <- any(layer_on_hover,layer_on_not_hover);
             model.visualization.frp.set_layer <+ layer;
-            eval layer ((l) model.error_visualization.frp.set_layer.emit(l));
+            eval layer ((l) model.error_visualization.set_layer(*l));
```

Comment:
We could also leave this layer constant, since error visualizations only appear on hover anyway